bgpd/bgp_vty.c

```diff
@@ -18189,6 +18207,12 @@ static void bgp_config_write_peer_global(struct vty *vty, struct bgp *bgp,
 	if (peergroup_flag_check(peer, PEER_FLAG_DONT_CAPABILITY))
 		vty_out(vty, " neighbor %s dont-capability-negotiate\n", addr);
 
+	/* capability fqdn*/
+	if (!peergroup_flag_check(peer, PEER_FLAG_CAPABILITY_FQDN))
```

Comment:
Does this flag work with peer groups? I think we need to add that to `peer_group2peer_config_copy()`.

Reply:
yes it works,  unitary tests are ok